src/main/java/net/sf/mzmine/parameters/parametertypes/filenames/FileNameParameter.java

```diff
@@ -99,7 +99,9 @@ public FileNameParameter cloneParameter() {
     public void setValueFromComponent(FileNameComponent component) {
         File compValue = component.getValue();
         if (extension != null) {
-            if (!compValue.getName().endsWith(extension))
+            if (!(compValue.getName().toUpperCase()
+                    .endsWith(extension.toUpperCase()) || compValue.getName()
+                    .toLowerCase().endsWith(extension.toLowerCase())))
```

Comment:
Yep, you are perfectly right ;)

I meant:

           if (!(compValue.getName().toUpperCase()
                   .endsWith(extension) || compValue.getName()
                   .toLowerCase().endsWith(extension)))

The idea being to prevent cases where case is mixed up like ".JpeG", ...

I'll fix it that way.